app/db/__init__.py

```diff
@@ -0,0 +1,17 @@
+import mysql.connector
+import os
+
+
+class DbConnection:
+    def __init__(self):
+        self.host = os.getenv('DB_HOST')
```

Comment:
Are you use able to use `current_app` object here to pass your `environment variables`?